npm-debug.log

```diff
@@ -0,0 +1,20 @@
+0 info it worked if it ends with ok
```

Comment:
Can you remove this file? We don't want extra log files in the repo if we don't have to.